src/components/notifications/notifications.service.ts

```diff
@@ -336,6 +357,59 @@ export function getSlackConfiguration(slackConfigId: number, isDeleteComponent?:
     })
 }
 
+export function getWebhookConfiguration(webhookConfigId: number): Promise<ResponseType> {
+    const URL = `${Routes.NOTIFIER}/channel/webhook/${webhookConfigId}`
+    return get(URL)
+}
+
+export function saveWebhookConfiguration(data): Promise<UpdateConfigResponseType> {
+    const URL = `${Routes.NOTIFIER}/channel`
+    const headerObj = {};
+    const headerPayload =  data.payload != '' ? JSON.parse(data.payload) : {}
+    data.header.forEach((element) => {
+        if (element.key != '') {
+            headerObj[element.key] = element.value
+        }
+    });
+    let payload = {
+        channel: 'webhook',
+        configs: [
+            {
+                configName: data.configName,
+                webhookUrl: data.webhookUrl,
+                header: headerObj,
+                payload: headerPayload,
+            },
+        ],
+    }
+    return post(URL, payload)
+}
+
+export function updateWebhookConfiguration(data): Promise<UpdateConfigResponseType> {
+    const URL = `${Routes.NOTIFIER}/channel`
+    const headerObj = {};
+    const headerPayload = data.payload != '' ? JSON.parse(data.payload) : {}
+    data.header.forEach((element) => {
+        if (element.key != '') {
+            headerObj[element.key] = element.value
+        }
+    });
+
+    let payload = {
+        channel: 'webhook',
+        configs: [
+            {
+                id: data.id,
+                configName: data.configName,
+                webhookUrl: data.webhookUrl,
+                header: headerObj,
+                payload: headerPayload,
+            },
+        ],
+    }
+    return post(URL, payload)
```

Comment:
we can directly use the value. no need to create variable

Reply:
done